satellite/repair/checker/checker.go

```diff
@@ -293,6 +296,14 @@ func (obs *checkerObserver) RemoteSegment(ctx context.Context, path metainfo.Sco
 			}
 		}
 
+		var segmentStart time.Time
+		if pointer.CreationDate.Before(pointer.LastRepaired) {
+			segmentStart = pointer.CreationDate
+		} else {
+			segmentStart = pointer.LastRepaired
+		}
+		mon.IntVal("checker_segment_irreparable_time_until_repair").Observe(int64(time.Since(segmentStart).Seconds()))
```

Comment:
this could be `segment_time_until_irreparable`

Reply:
Also nit - I think this could be improved by splitting into multiple lines like with `checker_segment_age`